lib/customs.js

```diff
@@ -14,6 +14,11 @@ module.exports = function (log, error) {
 
   // Perform a deep clone of payload and remove user password.
   function sanitizePayload(payload) {
+
+    if (! payload) {
+      return
```

Comment:
Turns out this is not really needed. I might have added it in for testing and forgot to remove :/